addons/point_of_sale/views/account_statement_view.xml

```diff
@@ -9,10 +9,22 @@
                     <page string="Point of Sale" attrs="{'invisible': [('type','not in',['bank','cash'])]}">
                         <group col="4" colspan="4">
                             <field name="journal_user"/>
-                            <field name="amount_authorized_diff"/>
+                            <field name="amount_authorized_diff" attrs="{'invisible': [('type','!=','cash')]}"/>
                         </group>
                     </page>
                 </xpath>
             </field>
         </record>
+        <record id="view_account_bank_journal_pos_user_form" model="ir.ui.view">
```

Comment:
Why you need this view? There is already a view for **account.journal**. See this: https://github.com/odoo/odoo/pull/15436/files#diff-693b40947e7766b1d64a05686dfac8fcR3 .